### ISSUE TITLE
Visibility issues in terminals with white background

### ISSUE THREAD

**Issue by curusarn** [cantino/mcfly#79]
This is how McFly looks like in a terminal with a white background. This one specifically is the gnome-terminal with Xterm color profile with black and white as foreground and background colors respectively. However, it is very likely that this will happen in almost any theme with a white background color. 

![mcfly-white](https://user-images.githubusercontent.com/10132717/79700467-0cd1c200-8296-11ea-9b86-649e9a1c624a.png)

The issue seems to be that you are using "color 15" for drawing the foreground text instead of using the foreground color. Here I changed the "color 15" to red in the terminal theme.

![mcfly-color15](https://user-images.githubusercontent.com/10132717/79700466-0c392b80-8296-11ea-84ca-ab33d1e5b8d5.png)

**Comment by cantino** [comment]
Thanks @curusarn. Can an app query for the foreground color and draw using that?

**Comment by curusarn** [comment]
The foreground color is the default. You change to different colors with different escape codes and then there is an escape code for resetting the colors.

From https://www.lihaoyi.com/post/BuildyourownCommandLinewithANSIescapecodes.html:
```
8 different colors:

Black: \u001b[30m
Red: \u001b[31m
Green: \u001b[32m
Yellow: \u001b[33m
Blue: \u001b[34m
Magenta: \u001b[35m
Cyan: \u001b[36m
White: \u001b[37m
Reset: \u001b[0m
```

If you are using a library to color the text then there should still be some kind of "default" or "reset" functionality.

**Comment by freamon** [comment]
Does "light mode" not solve this problem?  Using the MCFLY_LIGHT environment variable renders text visible on a white background.

**Comment by utdrmac** [comment]
MCFLY_LIGHT worked for me. I run a white background and don't see any issues with artifact drawing.

**Comment by DamienCassou** [comment]
I can reproduce the bug with:

```
$ env | grep MCFLY
MCFLY_SESSION_ID=TU1SeEwqMBZgxqaodqjt4aoq
MCFLY_KEY_SCHEME=emacs
MCFLY_LIGHT=TRUE
MCFLY_HISTORY=/tmp/mcfly.3APseccT
```

![2020-08-26-163613](https://user-images.githubusercontent.com/217543/91317438-59845700-e7ba-11ea-9244-5963c4a17422.png)

**Comment by szigya** [comment]
I have the same problem, because I use terminal with dark theme, but intellij with light. It would be nice to have either a color which is visible with both or use default colors

**Comment by EgidioCaprino** [comment]
I have the same problem.

```
$ env | grep -i mcfly
MCFLY_LIGHT=FALSE
MCFLY_SESSION_ID=0tyrS3WfN8wKhQgyTpfYJkkM
MCFLY_HISTORY=/tmp/mcfly.JGz25H95
MCFLY_HISTORY_FORMAT=zsh-extended
```

![image](https://user-images.githubusercontent.com/5156120/162585213-57eca9e9-fd6e-4476-957a-2b80a6f8c2d4.png)

**Comment by EgidioCaprino** [comment]
`MCFLY_LIGHT=FALSE` sets the light mode anyway. To use it on a dark background you have to remove that variable declaration.

**Comment by ulises-castro** [comment]
I had same issues, idk why
<img width="853" alt="image" src="https://user-images.githubusercontent.com/17864276/171296790-e089bce6-1c53-48b5-87db-0cd721994cc7.png">

Had someone solved it?

**Comment by acesaro** [comment]
> 
> nevermind I started using fzf..

I gave https://github.com/bnprks/mcfly-fzf a try and it works wonderfully to provide the best of both tools!

**Comment by ok-ttc** [comment]
I have the same issue.  My terminal switches from dark to light and vice versa according to the time of day. Visibility during the "light time" is problematic.

Is there a fix yet?

**Comment by ok-ttc** [comment]
> Does "light mode" not solve this problem? Using the MCFLY_LIGHT environment variable renders text visible on a white background.

No. In my case I switch themes depending on the time of day (or surrounding lighting). So depending on whether I use light or dark mode, I have visibility issues in the one or the other mode.
